lib/beaker-answers/answers.rb

```diff
@@ -25,8 +25,8 @@ class Answers
       :q_puppetdb_database_user                      => 'mYpdBu3r',
       :q_database_port                               => 5432,
       :q_puppetdb_port                               => 8081,
+      :q_classifier_database_name                    => 'pe-classifier',
       :q_classifier_database_user                    => 'DFGhjlkj',
-      :q_database_name                               => 'pe-classifier',
```

Comment:
This was probably a typo. Fixed here, and updated the reference in the 3.4 version file.